subsys/bootloader/provision/provision_flash.c

```diff
@@ -37,34 +48,57 @@ u32_t num_public_keys_read(void)
 	return p_provision_data->num_public_keys;
 }
 
+static const u32_t *get_key(u32_t key_idx)
+{
+	return &(p_provision_data->key_data[key_idx *
+			(CONFIG_SB_PUBLIC_KEY_HASH_LEN / 4 + 1)]);
+}
+
+#define INVALID_VAL 0xFFFF0000
```

Comment:
Either move this to the top, or right next to where it is used.

Reply:
It's used in the function directly below.